.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+  Thanks for submitting a pull request!
+  Please provide enough information so that others can review your pull request.
+-->
+
+**Summary**
+
+<!--
+  Explain the **motivation** for making this change.
+  What existing problem does the pull request solve?
+  Are there any linked issues?
+-->
```

Comment:
The task mentions "A link to associated RFC, when relevant". Here's a suggestion, feel free to adjust.

```suggestion
<!--
  Explain the **motivation** for making this change.
  What existing problem does the pull request solve?
  Are there any linked issues?
  
  If your contribution results from an RFC, please link it as well.
-->
```